src/test/java/space/obminyashka/items_exchange/end2end/UserFlowTest.java

```diff
@@ -114,35 +118,46 @@ void getChildren_Success_ShouldReturnUsersChildren() throws Exception {
     @WithMockUser(username = ADMIN_USERNAME)
     @DataSet("database_init.yml")
     @ExpectedDataSet(value = "children/update.yml", orderBy = {"created", "name", "birth_date"}, ignoreCols = "id")
-    void updateChild_Success_ShouldReturnHttpStatusOk() throws Exception {
+    void updateChild_success_shouldReturnHttpStatusOk() throws Exception {
         var validUpdatingChildDtoJson = getTestChildren(2018);
 
         sendDtoAndGetResultAction(put(USER_CHILD), validUpdatingChildDtoJson, status().isOk())
                 .andExpect(jsonPath("$", hasSize(2)));
     }
 
-    @Test
+    @ParameterizedTest
     @WithMockUser(username = ADMIN_USERNAME)
+    @MethodSource("listDataPassword")
     @DataSet("database_init.yml")
     @ExpectedDataSet(value = "user/changing_password_or_email_expected.yml", orderBy = "created",
             ignoreCols = {"password", "email", "lastOnlineTime", "updated"})
-    void updateUserPassword_WhenDataCorrect_Successfully() throws Exception {
-        UserChangePasswordDto userChangePasswordDto = new UserChangePasswordDto(CORRECT_OLD_PASSWORD, NEW_PASSWORD, NEW_PASSWORD);
-
-        MvcResult mvcResult = sendDtoAndGetMvcResult(put(USER_SERVICE_CHANGE_PASSWORD), userChangePasswordDto, status().isAccepted());
+    void updateUserPassword_shouldGetResponse
```

Comment:
No one ever split the method signature by the entrance bracket, it'd be much better to keep all parameters in line and to move throws Exception to the next line

Reply:
++